io.github.arunsivaramanneo.GPUViewer.yml

```diff
@@ -0,0 +1,119 @@
+id: io.github.arunsivaramanneo.GPUViewer
+runtime: org.gnome.Platform
```

Comment:
Is there any specific reasons to use the gnome runtime? as far as I can tell, the application uses Gtk only from GNOME. You should be using freedestkop runtime instead which has Gtk.

Reply:
FDO runtime is missing python-gobject.

Reply:
@TingPing should I switch to FDO runtime and and bundle everything needed, or Gnome runtime is fine?